### ISSUE TITLE
No lang attribute on html tag

### ISSUE THREAD

**Issue by p7g** [p7g/blog-generator#2]
> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
\- Chrome lighthouse
